src/ansi-c/padding.h

```diff
@@ -19,7 +19,7 @@ class struct_typet;
 class typet;
 class union_typet;
 
-mp_integer alignment(const typet &type, const namespacet &);
+mp_integer alignment(const typet &type, const namespacet &, bool in_packed);
```

Comment:
I'd make the namespace the last parameter, for consistency.

Reply:
Done.